tools/nl/embeddings/build_custom_dc_embeddings.py

```diff
@@ -142,9 +143,9 @@ def generate_embeddings_yaml(model_info: utils.ModelConfig,
       "version": 1,
       "indexes": {
           "custom_ft": {
-              "embeddings": embeddings_csv_handler.abspath(),
+              "embeddings_path": embeddings_csv_handler.abspath(),
```

Comment:
It'll be cool if at some point, changes to the yaml format don't require changes to the custom embeddings tool.

Reply:
I think update the generated yaml file should do that, so running code that generate v10 yaml file with <name>_v10.yaml. and the server would look for _v10.yaml and not read _v9.yaml (which can not parse).